reference/websockets/sails.io.js/socket.request.md

```diff
@@ -10,7 +10,30 @@ This function is very similar to `io.socket.get()`, `io.socket.post()`, etc. exc
 ### Usage
 
 ```js
-io.socket.request(options, function (data, jwr)){
+io.socket.request(url, reqData, function (resData, jwr)){
```

Comment:
1. Syntax error, one too many closing parentheses.
2. `request()` does not have that function signature: https://github.com/balderdashy/sails-generate-sails.io.js/blob/master/templates/sails.io.js#L847